### DISCUSSION TITLE
is:issue is:open examples does not covert working with binding collectionview or listview.

### DISCUSSION THREAD

**Discussion by BlockchainPunks** [CommunityToolkit/Maui#779] [General]
the community toolkit demo app demonstrates how to work with collections

MAUI demo code doesn't ?
![collectionview_mvvm](https://user-images.githubusercontent.com/97580331/203689906-2fc36ff1-4ed5-4f58-a8f7-0a5b4cf69eea.PNG)

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
